_pyceres/core/crs_matrix.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "_pyceres/core/wrappers.h"
+#include "_pyceres/helpers.h"
+#include "_pyceres/logging.h"
+
+#include <Eigen/Sparse>
+#include <ceres/ceres.h>
+#include <ceres/crs_matrix.h>
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
+
+namespace {
+py::tuple ConvertCRSToPyTuple(const ceres::CRSMatrix& crsMatrix) {
+  std::vector<int> rows, cols;
+  std::vector<double> values;
+
+  for (int row = 0; row < crsMatrix.num_rows; ++row) {
```

Comment:
The output rows will be in the format ``std::vector<int>`` (now in ``py::array_t<int>``). So there is no point to use size_t here since we need the ``row`` variable in the loop.

Reply:
Ok makes sense, thanks.